app/models/criteria.rb

```diff
@@ -1,13 +1,12 @@
 # frozen_string_literal: true
 # rubocop:disable Rails/FindEach
 class Criteria
-  ATTRIBUTES = CriteriaHash.reduce([]) do |attributes, criterion|
-    attributes | criterion[1].keys
-  end.map(&:to_sym).freeze
-  FUTURE_ATTRIBUTES = %i(
-    met_url na_placeholder na_suppress unmet
-    unmet_suppress
+  ATTRIBUTES = %i(
+    autofill category description details future major met_placeholder
+    met_suppress met_url_required minor na_allowed na_justification_required
+    na_placeholder rationale unmet_placeholder
   ).freeze
+  FUTURE_ATTRIBUTES = %i(met_url na_suppress unmet unmet_suppress).freeze
```

Comment:
Yes, good suggestion.  I will do that and group with david's suggestion.  It is a few more lines, but clearer overall.